### ISSUE TITLE
1.7.10 Autocraft close without showing number to create REPRODUCIBLE, WORKAROUND.

### ISSUE THREAD

**Issue by ronkkrop** [AppliedEnergistics/Applied-Energistics-2#2934]
- Minecraft Version:  1.7.10
- AE2 Version:  appliedenergistics2-rv2-stable-10
- Forge Version:  I'm not sure...Here are the list of jar's within the 1.7.10 folder:
ForgeMultipart-1.7.10-1.2.0.345-universal
ForgeRelocation-0.0.1.4-universal
ForgeRelocationFMP-0.0.1.2-universal

After spending a considerable of time trying to learn AE2, I discovered that when an item, any item, is placed within a specific slot in your inventory, the autocraft window will close when you should be presented with a followup window asking how many you'd like to create.

Also, when something is placed in this inventory slot, when viewing 'Crafting Status' within the ME Crafting Terminal, the client will definitely crash and occasionally, the server will crash.

The inventory slot is 2nd row down, 4th from the left.

I wouldn't believe this report if I had seen it for myself, so I've included a video, where I go through a few different permutations in an attempt to rule out chance.

https://vid.me/CGQg

Should you need any further information, I'd be more than happy to offer whatever assistance I can.

**Comment by mindforger** [comment]
i am sure this got addressed already, try out rv3 beta6 ... it also maybe fixed in dev for rv3 release but not pushed as "release" before rv4 hits beta! ... rv3 beta is MORE stable than rv2 stable

**Comment by ronkkrop** [comment]
looking at this other bug report, i doubt it:
https://forum.feed-the-beast.com/threads/ae2-clicking-autocrafting-closes-inventory.240633/

**Comment by mindforger** [comment]
do me a favor, go ingame, press and hold F3 + C for a few seconds until it crashes and post that forced crash report to have insight on your setup please :)

**Comment by ronkkrop** [comment]
Oh it's no bother.  I'm happy to help.  I wouldn't have posted here if I didn't.

This is on a server, so updating to the latest AE2 isn't an option.  To make things even stranger, an almost identical setup in a single player world of the same pack doesn't have the same problem.  So it has to be some odd interaction between mods or some kind of server-client issue.  I will continue to test and advise as I discover more details.  If you have any further suggestions, I'm more than happy to do anything required.

I have not tried rv3 or rv4 in single player.

**Comment by ronkkrop** [comment]
I forgot to attach the crash dump requested:
https://pastebin.com/HXRhbKZR

**Comment by mindforger** [comment]
okay that are way to many mods, when you setup your test, make sure you try to only run it using AE2 and not even optifine!
try adding following mods to check if one of them causes it when the bug is not present with AE2 alone anymore:

-optifine (always has an surprise sideeffect when not thinking of it)
-extracells!!!! (one of the biggest offenders in cases of multiple issues that time) you will have to find the fitting version for running RV3, i do not know where to find sorry!
-fastcraft (mostly the same as optifine)
-inventorytweaks (highly suspecetd for that particular issue)
-tcinventoryscan (dunno what it does, but reading something with inventory triggered an alert :P )

PS: WR-CBE (you really should not run this on 1.7.10 there are ways to permanently crash a server with that stuff on purpose, not just accidentially)

**Comment by ronkkrop** [comment]
Okay but first I'm going to run it with the exact same settings as the server.  I'd like to be able to reproduce it first.  If I get the same behavior as above, I'll try to pin it down with your suggestions.

**Comment by ronkkrop** [comment]
So I got the server up, tested with the exact same settings and was able to reproduce the issue, however, it is not the same inventory slot.  The inventory slot on the bottom row, 3rd from the left causes the autocrafter to close.

I will test again with your settings this evening, as it's too late to continue.

**Comment by ronkkrop** [comment]
Okay...a little more information.
1. Disabling optifine, extracells, inventorytweaks makes no difference on *this* world.  It does appear to work on a fresh world with a considerable delay in displaying the next window.

2. Fastcraft cannot be disabled as it's required by a few of the mods in the pack according to the pack author.  Fastcraft could be the culprit (though I have my doubts) but this obviously makes it very difficult to test.

3. I included the caught exception that the server reported in a pastebin for your review:
https://pastebin.com/2gX0k5Vj

Addendums: I didn't bother with tcinventoryscan as it's a thaumcraft addon that's really only active when the thaumometer is dragged over inventory items. There doesn't not appear to be any mention of it in the caught exception either.  WR-CBE:  it's mostly a group of friends that play on this server so nobody would deliberately crash the server, however, I am curious how somebody could accidentally crash the server so it's avoided.

**Comment by ronkkrop** [comment]
Fastcraft can now be ruled out as well.  I managed to get fastcraft disabled without breaking the server but still have the same end result.

**Comment by mindforger** [comment]
okay the log with the exceptions is very helpful

**Comment by ronkkrop** [comment]
Best of luck friend.  If you think of anything else to try, let me know I'll keep checking back.

**Comment by mindforger** [comment]
take the very latest AE2 version (rv4 beta) and retry it with 1.1.0.2 and grab the logs, then i can try to find it in the actual code because there are quiet a few differences in the code and it is harder to track it down then

**Comment by ronkkrop** [comment]
I don't think that's necessary any longer.  I went through all the mods that FTB Beyond and Pagemaster have in common and started disabling any that might have something to do with AE2.  After disabling 'Wireless Crafting Terminal', the issue is gone. 

I've now tried every combination of mods that tie into AE2 that I can and can definitively say that the issue only occurs when 'Wireless Crafting Terminal' mod is enabled.
